docs/Catalog/Qurator.md

```diff
@@ -0,0 +1,82 @@
+# Qurator Omni Beta
+
+`Qurator Omni` is an always-available AI assistant embedded directly into the
+Quilt web catalog. It allows users to interact with Quilt packages and search
+functionality through natural language. Qurator Omni leverages advanced models
+like Claude, integrated via Amazon Bedrock, enabling users to query, retrieve, and
+summarize data instead of having to click through the GUI.
+
+## What is Qurator Omni?
+
+Qurator Omni is designed to streamline interaction with Quilt data by offering a
+conversational interface. Instead of navigating through various tabs and menus,
+or learning complicated search syntax, users can opt into the Qurator feature to
+ask complex questions in plain language and receive structured, actionable
+responses.
+
+For example, users can ask for summaries of research on topics like "melanoma"
+or request key insights from a specific dataset.
+
+### Key Features
+
+- **Natural Language Queries**: Ask complex questions like “What are the latest
+  asthma treatments?” or “Summarize research on BRCA1 mutations.”
+- **Instant Summaries**: Quickly digest scientific papers, datasets, or reports
+  without reading everything.
+- **Fine-Grained Permissions with RAG**: Query only the data you're authorized
+  to access, ensuring compliance with strict organizational policies
+- **Secure Cloud Environment**: Work within your private AWS cloud, ensuring
+  data stays secure while using state-of-the-art AI models.
+
+## Getting Started
+
+To enable Qurator Omni:
+
+1. **Opt-In to Qurator**:  
+   - Install Release 1.55 or later of the Quilt Platform CloudFormation template.
+   - Set the `qurator` parameter to `true` in the CloudFormation template to
+     enable the Qurator chatbot.
+
+2. **Configure Claude Model**:  
+   - Login to the Amazon Bedrock console.
+   - Ensure that the Claude 3.5 Sonnet model is available in the same region as
+     your Quilt deployment. Check [Model support by AWS
+     Region](https://docs.aws.amazon.com/bedrock/latest/userguide/models-regions.html)
+     for details.
+   - Enable the model by configuring it within your Bedrock environment.
+   - Carefully monitor the model’s cost implications. The Claude model is
+     charged based on usage, so ensure that you have the necessary budget
+     allocated. Initial estimates are roughly a penny per page for complex documents,
```

Comment:
```suggestion
     allocated. Initial estimates are roughly a penny per page for complex documents.
```
or were you meaning to continue the sentence?

Reply:
also, "complex" or not -- doesn't actually matter

Reply:
I concede "complex" is somewhat grauitous, but AI does do a better job of interpreting complex items like tables.